test_data/nixpkgs_repository/maintainers_scripts_update.good.nix

```diff
@@ -29,8 +26,7 @@
     if attrSet == null then
       builtins.throw "Attribute path `${path}` does not exists."
     else
-      packagesWith
-        (name: pkg: builtins.hasAttr "updateScript" pkg)
```

Comment:
I think it's more readable when all the arguments are aligned vertically

Reply:
with this PR, it is depend on how the user format the code. In this case, because in the `bad.nix` , the code inline the first parameter.

One thing that the formatter make sure now is the `(` and `{` as the last argument in the function application.